catalog-info.yaml

```diff
@@ -436,11 +436,15 @@ spec:
       repository: elastic/logstash
       pipeline_file: ".buildkite/exhaustive_tests_pipeline.yml"
       provider_settings:
-        trigger_mode: none
-      cancel_intermediate_builds: true
-      skip_intermediate_builds: true
+        build_branches: true
+        build_pull_request_forks: false
+        build_pull_requests: false
+        build_tags: false
```

Comment:
conversely, I felt that we don't need to trigger this pipeline when we push tags.

Reply:
Agreed, not for tags.